src/host/outputStream.cpp

```diff
@@ -368,27 +368,23 @@ void ConhostInternalGetSet::SetWorkingDirectory(const std::wstring_view /*uri*/)
 // Arguments:
 // - noteNumber - The MIDI note number to be played (0 - 127).
 // - velocity - The force with which the note should be played (0 - 127).
-// - duration - How long the note should be sustained (in milliseconds).
+// - duration - How long the note should be sustained.
 // Return value:
 // - true if successful. false otherwise.
-void ConhostInternalGetSet::PlayMidiNote(const int noteNumber, const int velocity, const std::chrono::microseconds duration)
+void ConhostInternalGetSet::PlayMidiNote(const int noteNumber, const int velocity, const MidiDuration duration)
 {
-    // We create the audio instance on demand, and lock it for the duration
-    // of the note output so it can't be destroyed while in use.
-    auto& midiAudio = ServiceLocator::LocateGlobals().getConsoleInformation().GetMidiAudio();
-    midiAudio.Lock();
-
     // We then unlock the console, so the UI doesn't hang while we're busy.
     UnlockConsole();

```

Comment:
Same deal as the comments in `ControlCore::_terminalPlayMidiNote`. Just drop the "then".